### PR TITLE
Fix heresy within australian-snags.md

### PR DESCRIPTION
**Pull request by SirHall**

### CODE REVIEW

**Review by DGKSK8LIFE — APPROVED** [review]
Looks good. Not sure what other maintainers think though.

File: src/australian-snags.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ especially national events. Sometimes served at hardware chains.
 - Onions (4g per sausage)
 - Tomato sauce
```
Comment (Alhadis) [inline src/australian-snags.md:14]: You missed a spot:
```suggestion
- Dead horse
```
garn m8

Comment (SirHall) [reply src/australian-snags.md:14]: Uhh ackshually, *real* Aussies use 'bottle o' horse'. ;)

Comment (Alhadis) [reply src/australian-snags.md:14]: Yeah, nah, yeah.

**Pull request merged**